packages/carbon-graphs/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+* Changed
+  * Updated panning unit tests to supress warnings.
```

Comment:
nit: can we be a little more specific here?
"Updated panning unit tests to suppress console warnings from code items pending deprecation"